README.md

```diff
@@ -1,2 +1,23 @@
 # hw01
 Repository for homework 01
+
+#Katrina Turick: A Brief Biography  
```

Comment:
Make sure to leave a space between your heading specifier and text, otherwise it won't render.